plugins/luckgive/plugin/commands/luck_give_cmd.rb

```diff
@@ -0,0 +1,35 @@
+module AresMUSH
+  module LuckGive
+    class LuckGiveCmd
+      #luck/give <name>=<reason>
+      include CommandHandler
+      attr_accessor :target, :reason
+
+      def parse_args
+       args = cmd.parse_args(ArgParser.arg1_equals_arg2)
+       self.target = Character.named(args.arg1)
```

Comment:
Have added required args- probably something I need to do with all my code. I'm not sure what you're suggesting with with_a_char though. Help?

Reply:
I meant to make  

def parse_args
   self.target_name = whatever
end

def handle
    ClassTargetFinder.with_a_char(self.target_name) do |model|
         stuff
    end
end

Instead of trying to do Character.named in the parse method.

Reply:
Did this, would appreciate quick eyes on it just to make sure it looks right. Pretty sure it is.